libs/dapp-config/src/chains/substrate/index.tsx

```diff
@@ -62,21 +62,27 @@ export const chainsConfig: Record<number, ChainConfig> = {
         name: 'Tangle Explorer',
         url: process.env['USING_LOCAL_TANGLE']
           ? populateBlockExplorerStub('ws://127.0.0.1:9944')
-          : populateBlockExplorerStub('wss://rpc-archive.tangle.tools'),
+          : populateBlockExplorerStub('wss://testnet-rpc-archive.tangle.tools'),
```

Comment:
If this might change again in the future, it may be worth to make a constant and centralize it somewhere accessible.